packages/office-addin-dev-settings/src/publish.ts

```diff
@@ -0,0 +1,18 @@
+import * as childProcess from "child_process";
+
+/* global console */
+
+export async function publish(zipPath: string): Promise<void> {
```

Comment:
`publish` here is very generic. Would `publishToTeams` or `registerWithTeams` be better?

Reply:
Good point.  I can do this now, but note that eventually it will be some yet to be named shared endpoint not specific to Teams.